qa/tasks/rook.py

```diff
@@ -367,6 +367,30 @@ def rook_cluster(ctx, config):
                 'count': num_hosts,
                 'allowMultiplePerNode': True,
             },
+            'storage': {
+                'storageClassDeviceSets': [
+                    {
+                        'name': 'scratch',
+                        'count': num_devs,
+                        'portable': False,
+                        'volumeClaimTemplates': [
+                            {
+                                'metadata': {'name': 'data'},
+                                'spec': {
+                                    'resources': {
+                                        'requests': {
+                                            'storage': '10Gi'  # <= (lte) the actual PV size
+                                        }
+                                    },
+                                    'storageClassName': 'scratch',
```

Comment:
Are the PVs from a scratch storage class defined somewhere in the test already? If not, we won't be seeing OSDs created.

Reply:
https://github.com/ceph/ceph/blob/57c2ae079b42cde0e8f3deedc760183af81503b2/qa/tasks/kubeadm.py#L452